stdeb/util.py

```diff
@@ -1468,8 +1468,12 @@ def build_dsc(debinfo,
         fname = debinfo.udev_rules
         if not os.path.exists(fname):
             raise ValueError('udev rules file specified, but does not exist')
-        link_func(fname,
-                  os.path.join(debian_dir, '%s.udev' % debinfo.package))
+        if debinfo.with_python2:
+            link_func(fname,
+                      os.path.join(debian_dir, '%s.udev' % debinfo.package))
+        if debinfo.with_python3:
+            link_func(fname,
+                      os.path.join(debian_dir, '%s.udev' % debinfo.package3))
```

Comment:
Have you checked how this will behave when both with_python2 and with_python3 are true? Does it create issues if the udev rules are linked under both package names?

Reply:
I have tried this and it works fine; the udev rule file actually has to be present twice (with prefix `python-` and `python3-`) in that case to be picked up by both the Python 2 and 3 package.